src/main/resources/brapi/calls.json

```diff
@@ -14,7 +14,7 @@
           "1.0",
           "1.1",
           "1.2",
-          "1.3",
+          "1.3"
```

Comment:
Are we changing versions too in this PR, @gelli-leafnode ?

Reply:
2.0 must retain. Will revert this change. Thanks